src/components/Card/Card.test.tsx

```diff
@@ -1,37 +1,98 @@
-import { render } from '@testing-library/react';
 import React from 'react';
+import { render, screen } from '@testing-library/react';
 
 import { Card, CardProps } from './';
 
+import styles from './Card.module.scss';
+
 const DEFAULT_PROPS: CardProps = {
-  title: 'Title Text',
-  value: '123',
-  bottomText: 'Bottom Text'
+  title: '',
+  value: '',
+  bottomText: undefined
 };
 
+const MOCK_SKELETON_ID = 'mock-skeleton';
+
 jest.mock('../Skeleton', () => ({
-  Skeleton: () => <span>Skeleton</span>
+  Skeleton: () => <div data-testid={MOCK_SKELETON_ID} />
 }));
 
-const renderComponent = (props?: CardProps) => render(<Card {...props} />);
+describe('<Card />', () => {
+  test('should only render text in the correct order - title, value, bottom text', () => {
+    const { container } = render(<Card {...DEFAULT_PROPS} title={'1,'} value={'2,'} bottomText={'3'} />);
+    expect(container).toHaveTextContent('1,2,3');
+  });
 
-test('should work with synchronous data', () => {
-  const { getByText } = renderComponent(DEFAULT_PROPS);
-  expect(getByText('Title Text')).toBeInTheDocument();
-  expect(getByText('123')).toBeInTheDocument();
-  expect(getByText('Bottom Text')).toBeInTheDocument();
-});
+  describe('semantic tags', () => {
+    test('should render title as <label>', () => {
+      const { container } = render(<Card {...DEFAULT_PROPS} title={'mock title'} />);
+      const label = container.getElementsByTagName('label');
+      expect(label.length).toBe(1);
+      expect(label[0]).toHaveTextContent('mock title');
+    });
+  });
 
-test('should show skeleton when loading asynchronous data', () => {
-  const { getByText } = renderComponent({ ...DEFAULT_PROPS, value: { isLoading: true } });
-  expect(getByText('Skeleton')).toBeInTheDocument();
-});
+  describe('class names', () => {
+    test('should apply Title class to only the title element', () => {
+      const { container } = render(<Card {...DEFAULT_PROPS} title={'mock title'} />);
+      const title = container.getElementsByClassName(styles.Title);
+      expect(title.length).toBe(1);
+      expect(title[0]).toHaveTextContent('mock title');
+    });
+
+    test('should only apply Value class to only the value element', () => {
```

Comment:
Should this check other classes aren't applied? Its just the way I perceive the use of word _only_. Same with other tests.

Reply:
Hmm yes perhaps `only` isn't the right word to use here. I was checking that `styles.Value` is applied to the correct element and no others.

I just realised that there's a bit of a typo in here with the double `only` - it should be `should apply Value class to only the value element`.